src/test/java/gift/WishListRepositoryTest.java

```diff
@@ -0,0 +1,70 @@
+package gift;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import gift.exception.RepositoryException;
+import gift.model.WishList;
+import gift.service.WishListRepository;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+public class WishListRepositoryTest {
+
+    @Autowired
+    private WishListRepository wishListRepository;
+
+    @Test
+    void save() {
+        WishList wishListExpected = new WishList("test-account@test.com", 1, "product A", 1);
+        wishListRepository.save(wishListExpected);
+        assertThat(wishListRepository.findAll()).isNotNull();
+    }
+
+    @Test
+    void findByParameters() {
+        WishList wishListExpected_1 = new WishList("test-account-1@test.com", 1, "product A", 1);
+        WishList wishListExpected_2 = new WishList("test-account-1@test.com", 1, "product B", 1);
+        WishList wishListExpected_3 = new WishList("test-account-2@test.com", 2, "product A", 2);
+        WishList wishListExpected_4 = new WishList("test-account-2@test.com", 2, "product C", 1);
+        List<WishList> wishLists = new ArrayList<>();
+        List<WishList> memberId1_wishLists = new ArrayList<>();
+        List<WishList> memberId2_wishLists = new ArrayList<>();
+
+        wishLists.add(wishListExpected_1);
+        memberId1_wishLists.add(wishListExpected_1);
+        wishLists.add(wishListExpected_2);
+        memberId1_wishLists.add(wishListExpected_2);
+        wishLists.add(wishListExpected_3);
+        memberId2_wishLists.add(wishListExpected_3);
+        wishLists.add(wishListExpected_4);
+        memberId2_wishLists.add(wishListExpected_4);
+        wishListRepository.save(wishListExpected_1);
+        wishListRepository.save(wishListExpected_2);
+        wishListRepository.save(wishListExpected_3);
+        wishListRepository.save(wishListExpected_4);
+        assertAll(
+            () -> assertThat(wishListRepository.findAll()).isEqualTo(wishLists),
+            () -> assertThat(wishListRepository.findByMemberId(1)
+                .orElseThrow(() -> new RepositoryException("해당 유저의 위시 리스트가 존재하지 않습니다."))).isEqualTo(
+                memberId1_wishLists),
+            () -> assertThat(wishListRepository.findByMemberId(2)
+                .orElseThrow(() -> new RepositoryException("해당 유저의 위시 리스트가 존재하지 않습니다."))).isEqualTo(
+                memberId2_wishLists)
+        );
```

Comment:
findAll 과 findByMemberId 를 나눠서 테스트 해보는건 어떨까요?
하나의 기능만 테스트하면 테스트 하려는게 무엇인지 더 명확해질 것 같아요.

테스트를 분리하더라도 현재와 같은 테스트를 하나 남겨두고 싶으시다면 아래와 같은 형태로 바꿀 수 있을 것 같아요. 불필요한 로직을 제거해보면 어떨까요?

```suggestion
WishList wishListExpected_1 = new WishList("test-account-1@test.com", 1, "product A", 1);
        WishList wishListExpected_2 = new WishList("test-account-1@test.com", 1, "product B", 1);
        WishList wishListExpected_3 = new WishList("test-account-2@test.com", 2, "product A", 2);
        WishList wishListExpected_4 = new WishList("test-account-2@test.com", 2, "product C", 1);
        List<WishList> wishLists = new ArrayList<>();

        wishListRepository.save(wishListExpected_1);
        wishListRepository.save(wishListExpected_2);
        wishListRepository.save(wishListExpected_3);
        wishListRepository.save(wishListExpected_4);

        assertAll(
            () -> assertThat(wishListRepository.findAll()).isEqualTo(wishLists),
            () -> assertThat(wishListRepository.findByMemberId(1).size()).isEqualTo(2),
            () -> assertThat(wishListRepository.findByMemberId(2).size()).isEqualTo(2)
        );
```